system/Security/CheckPhpIni.php

```diff
@@ -134,12 +135,28 @@ public static function checkIni(): array
             'date.timezone'           => ['recommended' => 'UTC'],
             'mbstring.language'       => ['recommended' => 'neutral'],
             'opcache.enable'          => ['recommended' => '1'],
-            'opcache.enable_cli'      => [],
-            'opcache.jit'             => [],
-            'opcache.jit_buffer_size' => [],
+            'opcache.enable_cli'      => ['recommended' => '1'],
+            'opcache.jit'             => ['recommended' => 'tracing'],
+            'opcache.jit_buffer_size' => ['recommended' => '256', 'remark' => 'Adjust with your free space of memory'],
             'zend.assertions'         => ['recommended' => '-1'],
         ];
 
+        if ($argument === 'opcache') {
+            $items = [
+                'opcache.enable'                  => ['recommended' => '1'],
+                'opcache.enable_cli'              => ['recommended' => '1'],
+                'opcache.jit'                     => ['recommended' => 'tracing'],
+                'opcache.jit_buffer_size'         => ['recommended' => '256', 'remark' => 'Adjust with your free space of memory'],
+                'opcache.memory_consumption'      => ['recommended' => '512', 'remark' => 'Adjust with your free space of memory'],
+                'opcache.interned_strings_buffer' => ['recommended' => '64'],
+                'opcache.max_accelerated_files'   => ['recommended' => '40000', 'remark' => 'Find many files in your project (example: find your_project/ -iname *.php|wc -l)'],
+                'opcache.max_wasted_percentage'   => ['recommended' => '15'],
+                'opcache.validate_timestamps'     => ['recommended' => '0'],
+                'opcache.revalidate_freq'         => ['recommended' => '0'],
```

Comment:
Remove the recommended for `opcache.revalidate_freq`.
It will be ignored when `opcache.validate_timestamps` is disabled.